Vagrantfile

```diff
@@ -5,11 +5,11 @@
 require 'yaml'
 require 'rbconfig'
 
-params = YAML::load_file('./default.config.yaml')
- 
+params = YAML::load_file(File.join(__dir__, 'default.config.yaml'))
```

Comment:
Oh interesting, yeah I usually use this pattern in node and php - is this to ensure that some other action in the app changing the current working directory doesn't break loading the file? Is it just more idiomatic? I haven't seen the original approach fail - but this seems more defensive.